pkg/kubelet/cm/internal_container_lifecycle_windows.go

```diff
@@ -21,9 +21,31 @@ package cm
 
 import (
 	"k8s.io/api/core/v1"
+	utilfeature "k8s.io/apiserver/pkg/util/feature"
 	runtimeapi "k8s.io/cri-api/pkg/apis/runtime/v1"
+	"k8s.io/klog/v2"
+	kubefeatures "k8s.io/kubernetes/pkg/features"
+	"k8s.io/kubernetes/pkg/kubelet/winstats"
 )
 
 func (i *internalContainerLifecycleImpl) PreCreateContainer(pod *v1.Pod, container *v1.Container, containerConfig *runtimeapi.ContainerConfig) error {
+	if i.cpuManager != nil && utilfeature.DefaultFeatureGate.Enabled(kubefeatures.WindowsCPUAndMemoryAffinity) {
+		klog.Info("PreCreateContainer for Windows")
```

Comment:
nit: Do should this method log with some verbosity? maybe use V(2) or V(3)?

Reply:
I've left the more detailed log below, set it to V4, and removed this one.